### ISSUE TITLE
Convert literal to string Error

### ISSUE THREAD

**Issue by xkevin190** [Mesh4all/m4a-firmware#127]
#### Description

making the modifications so that the wifi module would take the data from the nvs, I was having problems when I set a data in the nvs it replaced the others.

#### Steps to reproduce the issue

For reproduce this issue you can  do simple printf to result optained of `nvs_key_to_name` for example:

#### Expected results

the code that was failing is this:

```c

#define literal_to_str(literal) #literal                /*!< Transform literal to string */
#define nvs_key_to_name(...) literal_to_str(__VA_ARGS_) /*!< A nonsense for the nvs component */
#define at_cmd_to_name(...) literal_to_str(__VA_ARGS__) /*!< Another one for the uart(at) component*

```

**Comment by luisan00** [comment]
please can you paste the error?

**Comment by luisan00** [comment]
Thanks a lot for the report! 
@xkevin190 Fixed into #128 , please try again compiling the related test
